src/Cpanel/PluginActions.php

```diff
@@ -21,4 +51,51 @@ public function applyDefaultSettings()
     {
         // Do nothing
     }
+
+    public function getConfig()
+    {
+        $this->getUserConfig();
+        $this->getComposerJson();
+
+        //Clone the config to manipulate
+        $config = array_merge(array(), self::$CONFIG);
```

Comment:
Basically I think you're right but since we're using `array_merge()` to _clone_ an array it doesn't matter.